lib/passkit/generator.rb

```diff
@@ -123,14 +121,18 @@ def generate_json_manifest
       File.write(@manifest_url, manifest.to_json)
     end
 
-    CERTIFICATE = Rails.root.join(ENV["PASSKIT_PRIVATE_P12_CERTIFICATE"])
-    INTERMEDIATE_CERTIFICATE = Rails.root.join(ENV["PASSKIT_APPLE_INTERMEDIATE_CERTIFICATE"])
```

Comment:
Constants are evaluated at load time, not runtime, so if those envs are nil, this will raise an error at startup of a rails app with `eager_load = true`. This bit us in production no less, because in staging we had everything configured correctly, and in local environment classes are not eager loaded, so it didn't manifest there.